### PR TITLE
Try to adopt Monace editor instead of Ace editor

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Dropped `react-ace` and brought `react-monaco-editor` instead ;) 
Screen Shot 2019-11-27 at 2 59 37 PM

^^ But I couldn't make the editor height fit in the container yet 😅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# In progress
```
Comment (sravan-s) [inline README.md:1]: Forgot to add this in prev. commits

File: package.json [hunk 2]
```diff
@@ -35,22 +35,22 @@
...
     "@fortawesome/react-fontawesome": "0.1.7",
     "@hot-loader/react-dom": "16.8.6",
-    "ace-builds": "1.4.7",
     "bootstrap": "4.3.1",
     "connected-react-router": "6.4.0",
     "functional-immutable": "1.0.1",
     "history": "4.9.0",
     "immutable": "4.0.0-rc.12",
     "lodash": "4.17.14",
+    "monaco-editor": "0.17.1",
```
Comment (sravan-s) [inline package.json:44]: https://github.com/microsoft/monaco-editor/issues/1623#issuecomment-540871567

File: src/components/Notebook/Paragraph.jsx [hunk 3]
```diff
@@ -43,64 +12,38 @@ const Container = styled.div`
...
-      }}
-      editorProps={{
-        $blockScrolling: Infinity,
-      }}
-    />
-  </Container>
-);
+}) => {
+  // todo - find better logic to set editor height
+  const height = ((editorText.split('\n').length) * LINE_HEIGHT) + PADDING;
```
Comment (sravan-s) [inline src/components/Notebook/Paragraph.jsx:28]: Hacky line height

Comment (AhyoungRyu) [reply src/components/Notebook/Paragraph.jsx:28]: Thanks! 🙏🏼

**Pull request merged**